tests/unit/test-api.c

```diff
@@ -332,14 +332,15 @@ main (void)
   jerry_api_object_t *global_obj_p, *obj_p;
   jerry_api_object_t *external_func_p, *external_construct_p;
   jerry_api_object_t *throw_test_handler_p;
+  jerry_api_object_t *err_obj_p = NULL;
   jerry_api_value_t res, args[2];
   char buffer[32];
 
-  is_ok = jerry_parse ((jerry_api_char_t *) test_source, strlen (test_source));
-  JERRY_ASSERT (is_ok);
+  is_ok = jerry_parse ((jerry_api_char_t *) test_source, strlen (test_source), &err_obj_p);
+  JERRY_ASSERT (is_ok && err_obj_p == NULL);
```

Comment:
Note: It's better practice to split and'ed asserts into two. In release, two asserts cost the same as one (i.e., nothing). However, if it comes to debugging, it's easier if the assert points exactly at source of failure. I.e., with the `&&` syntax, there will be a need for an extra round of execution to find out whether the `is_ok` or the `err_obj_p == NULL` part failed. With two asserts, that's not an issue.